apps/site/lib/site_web/templates/layout/_nav_mobile.html.eex

```diff
@@ -1,22 +1,51 @@
-<nav class="collapse row" id="navbar" role="navigation">
-  <ul class="mobile-nav">
-    <li class="mobile-nav-item search-mobile">
-      <%= Phoenix.HTML.Link.link to: search_path(@conn, :index), class: "mobile-nav-link" do %>
-        <span class="mobile-nav-item-header">
-          <%# we put the icon and mode name on one line to avoid the whitespace being collapsed. %>
-          <%= svg_icon(%SvgIcon{icon: :search, show_tooltip?: false}) %> <span>Search</span>
-        </span>
+<%= if Laboratory.enabled?(@conn, :nav_redesign) do %>
+  <!-- New nav -->
+  <nav class="m-menu" id="navmenu">
+    <div class="m-menu__cover"></div>
+    <!-- Do we need to use ul for access? -->
```

Comment:
I've been thinking about how to best structure these submenus, and there are several options -- from an accessibility standpoint it's most important that the menu structures can be understood programmatically, and nested unordered lists or a list of `<nav>` elements (one nav for each submenu, each nav having a distinct `aria-label`) both can work great for that. [Some more about that here](https://www.w3.org/WAI/WCAG21/Techniques/html/H97#examples).

Reply:
Ultimately we want to make sure each section is accessibly labelled, [example here](https://inclusive-components.design/menus-menu-buttons/#tablesofcontent) (the whole page is a neat read!)

Reply:
How will converting the navs to accordions impact the question of best-labeling?  Not sure how much I should do on this piece because the components will be replaced in the accordion ticket

Reply:
@hannahpurcell whatever labels you have here will likely have to be transferred over to work with the accordion markup.